content/chat.adoc

```diff
@@ -81,10 +81,10 @@ We currently offer the following cloak levels:
 
 To obtain a cloak:
```

Comment:
I would delete the cloak section tbh

Reply:
just for the record, https://gitter.im/jenkinsci/platform-sig
group members (olblak and I are primary group members, nobody is secondary yet) goes to #libera-community and go "Can $name have cloak of jenkins/whatever"

Reply:
Restored the cloak section and updated the third bullet to reflect the process described above.